src/ad-hoc-visualizations/needs-review/visualization.tsx

```diff
@@ -23,10 +25,14 @@ export const NeedsReviewAdHocVisualization: VisualizationConfiguration = {
         toggleLabel: 'Show areas needing review',
         linkToDetailsViewText: 'List view and filtering',
     },
-    chromeCommand: '01_toggle-issues',
     launchPanelDisplayOrder: 6,
     adhocToolsPanelDisplayOrder: 6,
-    getAnalyzer: () => null,
+    getAnalyzer: provider =>
+        provider.createBaseAnalyzer({
+            key: AdHocTestkeys.Color,
```

Comment:
We should probably refer to the needs review keys/visualization types in this code.

Reply:
Agreed, sorry, changing that